.travis.yml

```diff
@@ -1,27 +1,25 @@
+sudo: required
+dist: trusty
 language: ruby
+before_install: gem install -N bundler -v '~> 1.11'
```

Comment:
It is semantic versioning meaning "version 1.X.Y where X is at least 11" - it ends up installing the latest bundler right now. This might catch errors with newer versions, but since we don't have Gemfile.lock either I think it is ok.